src/js/containers/search/filters/cfda/CFDAListContainer.jsx

```diff
@@ -0,0 +1,180 @@
+            };
+
+            this.cfdaSearchRequest = SearchHelper.fetchCFDA(cfdaSearchParams);
+
+            this.cfdaSearchRequest.promise
+                .then((res) => {
+                    const data = res.data.results;
+                    let autocompleteData = [];
+                    const search = new Search('program_number');
+                    // ordering by prefix if there are matches returned that begin w/ the exact text
+                    search.indexStrategy = new PrefixIndexStrategy();
+                    search.addIndex(['program_number']);
+                    search.addIndex(['program_title']);
+                    search.addDocuments(data);
+                    const results = search.search(this.state.cfdaSearchString);
+                    const improvedResults = slice(results, 0, 10);
+
+                    // Filter out any selected CFDA that may be in the result set
+                    const selectedCFDA =
+                    this.props.selectedCFDA.toArray().map((cfda) => omit(cfda, 'identifier'));
+                    if (improvedResults && improvedResults.length > 0) {
+                        autocompleteData = differenceWith(improvedResults, selectedCFDA, isEqual);
+                    }
+                    else {
+                        autocompleteData = improvedResults;
+                    }
+
+                    this.setState({
+                        noResults: autocompleteData.length === 0
+                    });
+
+                    // Add search results to Redux
+                    this.props.setAutocompleteCFDA(autocompleteData);
+                })
+                .catch((err) => {
+                    if (!isCancel(err)) {
+                        this.setState({
+                            noResults: true
+                        });
+                    }
+                });
+        }
+        else if (this.cfdaSearchRequest) {
+            // A request is currently in-flight, cancel it
+            this.cfdaSearchRequest.cancel();
+        }
+    }
+
+    clearAutocompleteSuggestions() {
+        this.props.setAutocompleteCFDA([]);
+    }
+
+    handleTextInput(cfdaInput) {
+        // Clear existing cfdas to ensure user can't select an old or existing one
+        this.props.setAutocompleteCFDA([]);
+
+        // Grab input, clear any exiting timeout
+        const input = cfdaInput.target.value;
+        window.clearTimeout(this.timeout);
+
+        // Perform search if user doesn't type again for 300ms
+        this.timeout = window.setTimeout(() => {
+            this.queryAutocompleteCFDA(input);
+        }, 300);
+    }
+
+    render() {
+        return (
+            <Autocomplete
+                {...this.props}
+                values={this.state.autocompleteCFDA}
+                handleTextInput={this.handleTextInput}
+                onSelect={this.props.selectCFDA}
+                placeholder="eg: 10.553 - School Breakfast Program"
```

Comment:
Nitpick: can we change this to `e.g.,` like in Budget Categories _or_ change Budget Categories to match this `eg:`.